OpenChange/MAPIStoreGCSFolder.h

```diff
@@ -41,6 +41,7 @@
 
 /* synchronisation */
 - (BOOL) synchroniseCache;
+- (BOOL) synchroniseCacheForNameInContainer: (NSString *) nameInContainer;
```

Comment:
In Objective-C, the name of the parameters are part of the method signature, so `nameInContainer` here is redudant. I'd suggest to change the method signature to:
```Objective-C
synchroniseCacheFor: (NSString *) nameInContainer;
```